integration_annotation_manager.py

```diff
@@ -1,4 +1,4 @@
-#%%
+# %%
```

Comment:
can we clean this integration up a bit because it is not clear what is suppose to happen here

Reply:
yes, but im not doing it right now